### DISCUSSION TITLE
Disable restic maintenanceFrequency

### DISCUSSION THREAD

**Discussion by bdietrich12** [vmware-tanzu/velero#3907] [General discussions]
Hello,

i should like disable the restic maintenanceFrequency on a resticRespository, because it's run by multiple velero/restic on different kubernetes cluster.

I read the documentation, we can precise when run the maintenance like :

spec:
backupStorageLocation: restic-tests
maintenanceFrequency: 168h0m0s

I try ro use :

maintenanceFrequency: Never
maintenanceFrequency: never
maintenanceFrequency: forget

But i can't sync with restic repository...

maintenanceFrequency: 0h0m0s

It run the maintenance anyways.

Is it possible to disable the restic maintenance, because the maintenance (prune command) is running during the velero's backups.

The prune commande lock the restic repositories and velero is not able to performs backups.

**Comment by zubron** [comment]
As mentioned in https://github.com/vmware-tanzu/velero/issues/3909#issuecomment-874996332, it is not possible to disable the restic maintenance frequency. As a workaround, you could try setting it to a very long interval to avoid the maintenance being run (the maximum duration represented by a `time.Duration` is approximately 290 years), however this may have performance implications as your restic repository grows.
